compiler/src/iree/compiler/Dialect/Flow/IR/FlowOps.cpp

```diff
@@ -68,14 +68,13 @@ static LogicalResult verifyDispatchWorkload(
              << workgroupCount.getNumArguments()
              << " arguments but dispatch provides " << workload.size();
     }
-    for (auto it : llvm::enumerate(llvm::zip_equal(
+    for (auto &&[index, types] : llvm::enumerate(llvm::zip_equal(
```

Comment:
nit: I don't think you need `&&` here and elsewhere with `enumerate`. This is because, confusingly, `&&` applies to the destructured intermediate value and not (directly) to `index` and `types`. We have a couple tests for this in `llvm/unittests/ADT/IteratorTest.cpp`